src/site/content/en/patterns/web-vitals-patterns/images/img-tag/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: pattern
+title: <img> tag
+description: Use the width and height attributes to avoid layout shifts
+date: 2021-08-17
+updated: 2021-08-17
```

Comment:
You can use height: XXX (e.g. height: 400) field in the frontmatter to enforce the demo frame to be bigger, if you want.